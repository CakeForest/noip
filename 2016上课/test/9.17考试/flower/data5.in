1000 1000
1 455 283
1 701 115
1 786 672
2 313 355
2 53 912
1 945 833
2 321 756
1 982 647
2 144 196
1 161 130
2 173 450
1 659 45
1 253 440
1 510 155
2 186 649
2 22 474
1 787 19
2 391 958
1 477 626
1 824 315
1 372 875
2 70 833
2 297 518
2 270 773
2 192 668
2 102 480
2 627 802
2 527 625
2 23 924
1 181 62
2 432 505
2 31 725
1 222 143
1 900 65
2 360 413
1 170 271
2 711 833
1 667 897
2 140 550
1 624 696
2 125 576
1 302 659
2 466 678
2 484 851
1 119 465
1 87 801
1 10 927
2 170 315
1 43 228
1 109 165
1 565 87
2 474 577
2 627 629
1 520 424
1 123 963
1 261 738
2 264 525
1 116 203
1 11 327
2 411 547
2 520 790
1 763 189
1 662 852
2 713 900
1 365 579
2 200 477
1 303 440
1 324 358
2 108 113
2 801 850
1 993 429
1 540 406
2 704 835
1 350 73
2 485 556
1 357 627
1 337 358
2 361 869
1 617 23
1 696 718
2 41 423
2 229 565
2 296 932
2 53 962
1 654 735
1 369 458
1 607 964
2 635 911
2 675 848
1 142 224
1 741 512
2 459 825
2 626 870
1 783 206
1 279 399
2 193 734
2 534 556
2 176 705
1 881 549
1 641 414
2 142 855
1 877 612
1 752 679
2 296 673
1 875 314
1 610 819
2 112 932
2 169 831
1 685 489
1 589 498
1 353 146
1 740 652
1 335 259
2 192 605
1 503 182
2 608 775
1 549 998
1 627 562
2 129 541
1 174 814
2 77 215
2 213 992
1 392 602
2 428 670
2 75 84
1 970 499
2 604 847
2 221 663
1 10 364
2 240 489
1 619 543
2 591 704
1 750 233
2 539 975
2 98 422
2 584 648
2 864 913
2 545 712
1 769 679
1 985 520
2 865 944
1 508 246
1 601 871
2 132 472
1 570 88
2 103 901
2 527 600
2 15 565
1 347 544
1 637 944
2 49 463
2 342 588
1 221 61
1 888 955
1 949 691
2 430 620
1 536 68
2 35 226
2 38 853
2 224 748
2 257 359
1 955 945
1 411 727
2 1 355
2 496 584
2 342 964
2 142 913
1 72 949
1 173 607
2 404 705
1 375 813
2 36 565
1 814 142
1 652 257
1 482 839
2 15 131
1 625 842
2 186 637
1 662 651
1 353 894
1 8 453
1 454 234
2 303 634
1 124 149
2 109 213
1 80 201
1 50 859
2 264 361
2 643 676
2 561 902
2 282 948
2 220 674
1 831 924
2 259 878
1 971 620
2 781 945
1 685 393
2 589 698
1 37 939
1 234 462
1 959 962
2 269 515
2 58 869
1 264 972
2 215 555
2 39 330
1 82 289
1 710 86
1 380 775
2 541 951
2 110 679
1 788 74
2 132 956
2 8 113
2 723 790
2 28 184
1 71 201
2 71 974
2 153 867
2 168 825
1 650 630
1 693 310
1 116 81
2 528 667
2 421 864
2 816 826
1 666 727
2 681 964
1 21 687
1 64 722
2 415 902
2 124 941
2 423 762
2 268 806
1 907 603
2 12 481
1 114 631
2 84 556
1 996 294
1 345 55
1 491 249
1 114 132
2 722 958
1 52 996
2 238 479
2 866 918
2 486 498
1 633 463
1 146 23
1 925 38
2 458 602
2 98 830
1 278 601
2 352 448
1 315 541
2 567 762
1 418 398
2 828 851
1 449 231
2 229 658
1 560 941
2 162 655
2 361 792
1 146 399
1 188 947
2 638 663
2 515 521
2 528 615
1 905 571
1 962 558
2 524 549
2 330 923
1 925 334
1 336 738
2 278 393
1 164 715
2 135 949
2 4 337
2 623 664
1 568 609
2 85 152
2 194 652
1 396 827
1 640 250
1 943 820
2 289 941
2 565 805
2 216 450
2 64 609
1 74 894
2 300 695
2 161 589
1 358 969
2 268 426
1 774 423
2 552 910
1 495 392
1 364 875
1 460 256
1 821 973
1 577 548
2 195 605
1 343 951
1 12 482
1 428 440
1 967 763
1 908 416
2 434 759
1 319 487
1 806 946
1 367 701
1 532 788
1 447 975
2 222 283
2 376 583
1 982 724
1 220 777
2 182 856
1 324 926
1 969 678
2 534 677
1 991 69
1 828 784
2 426 871
2 612 703
2 408 545
1 238 186
2 313 443
2 128 850
2 149 650
1 869 455
2 267 465
2 634 793
1 830 973
2 177 442
2 702 770
1 590 382
2 23 237
2 169 595
2 42 310
1 926 59
2 528 670
2 213 258
1 286 784
1 472 611
1 841 435
1 867 504
2 881 938
2 614 750
1 661 459
2 756 807
1 435 490
2 75 586
1 360 834
1 928 49
1 840 434
1 810 736
2 837 892
1 352 329
2 244 794
1 647 253
1 208 536
1 243 498
2 15 841
2 100 812
1 851 524
1 891 634
1 990 855
2 780 919
1 544 932
1 899 488
2 483 538
1 252 194
2 560 834
1 785 498
2 540 805
2 210 392
2 578 979
2 73 277
2 497 620
1 790 277
1 159 579
1 159 490
2 72 924
1 967 9
1 503 478
1 196 608
1 611 723
2 56 761
1 683 164
1 452 933
2 813 954
1 460 397
2 599 904
1 198 681
1 584 388
1 6 518
1 882 242
2 523 758
2 95 621
1 678 917
1 58 580
2 7 750
2 81 995
1 753 677
2 565 784
2 172 608
2 514 929
1 191 56
2 748 922
2 144 986
1 517 578
2 874 916
2 469 912
1 91 694
2 857 949
1 236 53
2 226 487
1 183 956
1 97 181
2 65 513
2 78 578
1 611 141
2 170 445
2 489 750
2 333 865
1 7 283
1 367 897
1 810 883
2 187 231
2 321 479
1 447 352
1 276 647
2 189 422
1 455 487
1 860 615
2 348 777
2 431 861
1 197 456
1 821 753
1 21 282
2 124 947
1 376 136
1 998 860
1 922 254
2 643 888
2 232 747
1 678 927
1 961 802
2 363 855
1 561 574
2 274 473
1 181 354
2 110 699
2 172 465
2 112 981
1 247 382
1 805 672
1 989 33
1 431 166
1 206 294
1 206 949
2 166 396
2 20 694
2 109 788
1 978 970
2 15 626
1 906 169
1 118 98
1 785 200
1 420 200
1 813 272
2 85 318
1 267 332
2 186 444
2 360 827
1 152 432
2 268 693
2 311 337
1 406 678
1 413 23
1 537 543
1 355 389
2 181 647
2 584 987
2 217 493
2 447 482
2 104 753
1 525 96
2 781 964
1 656 107
2 80 593
1 411 869
2 251 968
1 768 80
1 933 532
2 259 663
2 95 936
2 720 874
2 680 976
2 71 725
1 156 560
1 905 833
1 562 376
2 80 962
1 202 798
1 340 81
1 493 59
1 474 547
2 97 880
2 72 400
2 666 955
2 481 588
1 396 316
2 9 12
1 762 456
2 21 742
1 248 513
1 717 369
1 290 651
2 169 759
2 303 640
2 105 199
2 661 681
1 33 754
2 42 987
2 83 420
1 244 719
2 229 652
1 470 770
2 47 594
2 276 326
2 540 679
1 710 589
2 221 945
1 589 184
2 779 978
1 135 263
2 33 196
1 779 936
2 790 962
2 1 105
2 567 669
1 457 672
1 597 546
1 844 839
1 28 564
1 723 802
1 601 605
2 197 692
2 301 363
2 565 721
2 445 610
2 22 741
1 15 152
2 393 738
2 608 882
1 707 823
2 144 338
1 154 340
1 225 624
1 981 725
1 223 734
1 846 131
2 445 805
1 489 751
1 135 964
2 209 630
1 737 909
1 372 921
2 734 855
2 56 606
1 382 76
2 432 741
1 279 780
2 667 836
2 118 737
1 577 120
2 91 556
2 60 901
2 136 432
1 907 876
1 719 75
1 41 477
2 290 329
1 591 73
2 490 787
2 53 893
2 681 903
2 176 479
2 139 468
1 639 84
2 621 993
1 838 723
1 399 582
1 23 892
2 243 834
2 702 707
1 687 142
1 637 892
2 400 816
1 935 163
1 877 411
1 189 261
2 663 874
1 566 196
1 588 39
2 245 467
2 189 867
1 547 64
1 99 618
2 203 226
1 570 52
1 967 459
1 531 406
1 975 820
1 495 532
1 803 592
1 450 916
2 272 791
2 133 775
1 300 570
1 585 79
2 333 893
1 750 104
2 271 722
2 338 990
2 47 258
1 479 659
1 990 684
1 422 569
2 8 135
2 194 742
2 178 188
1 941 358
2 345 469
1 964 914
1 385 62
2 462 504
2 69 102
1 552 530
1 730 150
1 49 845
1 363 66
1 470 774
2 511 747
2 398 498
2 352 679
2 43 522
1 834 564
1 240 23
2 492 651
1 616 478
1 220 179
2 348 798
2 635 857
2 662 902
1 770 421
1 841 274
1 917 889
2 698 892
2 65 749
2 619 932
2 3 130
1 589 632
1 172 631
1 295 408
1 490 682
1 71 178
1 643 460
1 37 634
2 274 630
2 203 782
2 290 662
2 151 997
2 717 890
1 566 704
2 659 661
2 386 651
2 601 840
2 283 497
1 421 59
2 581 787
2 287 999
1 138 980
1 461 531
1 540 206
1 622 460
1 98 761
2 481 614
2 493 596
2 538 746
1 476 539
2 371 917
1 758 753
2 65 255
1 34 402
2 393 548
2 193 792
2 103 137
1 694 212
1 498 628
2 248 879
2 149 931
2 598 875
2 436 599
2 443 731
1 434 862
2 816 967
2 704 866
2 223 308
1 667 793
1 930 470
2 90 814
2 604 743
2 195 474
1 589 159
1 889 810
2 40 818
1 26 976
2 66 503
1 570 841
1 641 396
1 109 652
2 83 85
1 686 251
2 128 381
1 862 278
1 872 566
1 428 886
1 478 890
2 265 925
1 778 46
2 520 855
2 134 773
2 336 675
1 737 2
1 590 975
1 80 72
2 145 282
2 573 718
1 998 38
2 162 717
1 47 540
1 231 104
2 839 858
2 394 645
2 272 645
2 72 862
2 238 480
2 542 608
2 125 277
1 78 402
1 736 170
1 138 940
2 426 663
2 575 724
2 700 961
1 448 3
2 16 684
2 507 993
1 259 945
2 58 643
1 119 678
2 41 891
1 915 624
1 841 930
2 536 615
2 700 759
1 241 94
2 227 448
1 324 225
1 885 134
1 862 171
2 84 909
1 85 924
1 193 25
2 412 765
2 407 677
1 790 705
1 621 892
2 190 734
1 86 999
1 424 622
2 254 376
2 108 927
2 68 366
1 600 439
2 290 318
1 556 340
1 478 633
1 239 788
1 827 21
1 441 989
1 2 643
2 104 946
1 833 510
1 533 762
1 385 805
1 260 843
2 343 620
1 113 188
2 591 934
2 656 761
1 959 412
2 370 738
1 7 508
1 101 952
2 441 958
1 412 14
2 60 93
1 676 403
2 266 373
2 275 528
1 469 762
2 107 729
1 953 452
1 519 571
1 494 408
2 383 440
1 607 410
1 471 361
2 654 947
1 229 524
2 766 887
1 470 633
2 522 666
2 160 553
2 91 259
1 629 688
1 45 318
1 452 389
1 833 467
1 631 920
1 776 268
1 320 315
2 7 469
1 842 723
2 260 463
2 630 793
2 43 376
1 117 627
1 171 621
1 154 965
1 664 694
2 0 212
1 476 552
2 877 943
2 361 385
1 144 435
2 504 563
1 308 889
2 123 430
1 346 75
2 318 981
2 292 591
1 461 124
2 330 461
1 291 370
1 117 180
2 595 914
2 867 936
1 909 454
2 503 981
2 816 883
2 385 402
1 681 158
2 310 866
2 171 477
2 238 764
2 47 115
1 657 312
2 53 171
1 530 741
2 320 790
2 586 998
1 631 490
1 610 389
1 259 920
2 119 609
2 300 716
1 458 854
1 806 594
2 272 854
1 54 295
2 146 229
2 428 559
2 421 688
1 685 34
1 261 330
1 982 209
1 878 507
2 887 897
1 783 470
1 266 796
2 509 557
2 53 134
2 258 593
2 466 693
2 94 140
2 174 512
2 551 695
2 292 492
2 530 613
1 389 711
1 786 810
1 182 841
2 120 393
1 214 963
2 194 807
2 11 713
2 664 693
2 762 995
2 909 946
1 624 769
2 235 954
2 240 633
1 577 128
2 142 340
1 558 401
1 935 155
1 295 832
2 623 922
2 29 211
2 754 880
2 293 321
2 443 528
1 671 556
1 375 453
2 640 934
1 834 987
1 765 540
1 262 313
1 987 732
1 248 558
1 450 171
1 293 302
2 44 332
2 4 75
1 66 27
1 623 802
2 380 441
2 659 918
2 329 628
1 613 855
2 569 666
2 577 837
1 292 116
2 448 872
2 153 948
2 197 508
1 993 412
2 202 781
1 256 829
2 23 965
2 253 715
2 478 684
2 833 840
2 900 968
1 885 634
1 32 282
1 872 911
1 306 902
2 471 860
2 137 343
1 995 234
2 603 942
2 34 879
1 530 594
2 234 258
1 191 907
2 188 227
2 224 484
2 224 435
1 867 583
2 5 146
1 540 590
2 759 850
1 677 957
2 42 300
2 143 225
2 16 615
1 368 7
2 335 698
2 23 313
1 457 619
1 86 340
1 43 847
1 280 912
2 358 847
1 873 991
2 413 454
1 450 473
2 641 646
2 213 824
2 403 695
1 67 994
2 79 640
2 125 534
1 907 1
