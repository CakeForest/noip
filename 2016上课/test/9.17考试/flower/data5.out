455 737
738 852
853 999
0
458
Can not put any one.
0
Can not put any one.
0
161 290
118
659 703
253 737
738 892
397
12
893 911
308
477 958
Can not put any one.
372 476
462
0
0
0
0
0
0
91
181 242
0
62
222 364
900 924
5
170 578
0
667 899
381
624 666
26
302 623
213
173
119 747
87 851
10 86
146
170 315
Can not put any one.
Can not put any one.
104
3
520 629
474 519
Can not put any one.
262
264 466
467 525
137
243
763 790
662 762
188
411 900
278
303 477
Can not put any one.
6
50
Can not put any one.
801 850
132
704 776
72
485 835
Can not put any one.
509
617 639
696 869
211
0
260
30
654 962
369 653
Can not put any one.
277
0
142 365
741 911
261
45
783 870
366 782
542
0
17
Can not put any one.
641 734
249
Can not put any one.
752 855
0
Can not put any one.
610 751
323
0
685 932
589 684
353 498
Can not put any one.
335 588
271
503 605
168
608 775
Can not put any one.
39
174 541
42
777
392 992
243
0
Can not put any one.
177
36
41 404
165
619 847
86
Can not put any one.
271
142
0
0
0
769 975
Can not put any one.
80
508 753
754 944
0
Can not put any one.
394
0
83
347 890
891 901
117
125
221 281
Can not put any one.
Can not put any one.
32
536 603
6
356
0
0
Can not put any one.
411 853
5
89
465
0
72 964
Can not put any one.
302
404 705
494
Can not put any one.
Can not put any one.
482 565
0
Can not put any one.
156
Can not put any one.
353 637
8 352
Can not put any one.
332
303 451
105
109 547
548 634
98
34
308
245
44
831 948
48
Can not put any one.
67
685 945
14
220 698
Can not put any one.
Can not put any one.
247
565
264 869
292
19
82 370
Can not put any one.
380 555
411
422
788 861
79
59
0
0
71 271
219
0
0
650 974
Can not put any one.
116 196
18
197
0
666 864
284
21 803
804 964
488
330
0
0
907 941
103
114 744
443
Can not put any one.
345 399
491 906
114 245
237
52 958
242
53
13
866 918
238 260
Can not put any one.
110
391
278 830
97
352 448
196
567 762
24
828 851
381
560 658
96
137
146 544
545 792
26
7
88
Can not put any one.
Can not put any one.
4
469
Can not put any one.
336 923
110
278 393
804
46
0
568 949
0
85
396 652
Can not put any one.
Can not put any one.
546
0
0
0
74 941
396
139
358 695
69
Can not put any one.
359
552 910
364 426
Can not put any one.
Can not put any one.
Can not put any one.
242
343 605
12 342
Can not put any one.
Can not put any one.
Can not put any one.
326
434 759
Can not put any one.
Can not put any one.
Can not put any one.
Can not put any one.
62
208
Can not put any one.
222 583
675
324 856
Can not put any one.
144
Can not put any one.
Can not put any one.
302
0
18
238 507
131
193
0
869 871
0
0
830 868
0
0
590 829
105
6
0
Can not put any one.
75
0
286 670
Can not put any one.
Can not put any one.
Can not put any one.
58
137
661 938
52
614 807
301
360 586
Can not put any one.
Can not put any one.
Can not put any one.
56
352 892
443
647 794
208 646
Can not put any one.
642
0
Can not put any one.
Can not put any one.
Can not put any one.
78
544 919
Can not put any one.
0
252 445
275
785 834
37
141
174
0
0
790 979
159 789
Can not put any one.
766
Can not put any one.
503 924
196 502
Can not put any one.
566
683 761
452 682
142
813 954
306
198 904
Can not put any one.
6 197
Can not put any one.
236
428
678 758
95 674
741
245
753 995
32
0
145
191 246
0
113
517 986
43
357
91 784
33
785 837
262
226 949
Can not put any one.
423
65
Can not put any one.
0
172
115
7 289
367 865
Can not put any one.
45
113
447 479
290 446
191
Can not put any one.
Can not put any one.
355
84
197 652
821 861
187 820
824
376 511
Can not put any one.
922 947
0
136
678 921
Can not put any one.
178
561 855
0
181 534
507
0
282
247 628
805 981
Can not put any one.
629 794
206 804
Can not put any one.
191
388
94
Can not put any one.
5
Can not put any one.
118 215
785 788
420 619
Can not put any one.
98
267 784
178
383
152 583
316
0
406 827
Can not put any one.
Can not put any one.
355 405
380
340
0
0
29
525 620
0
656 762
69
411 987
558
768 847
933 968
0
84
0
40
0
156 715
905 976
716 904
807
202 962
Can not put any one.
Can not put any one.
Can not put any one.
679
0
75
0
396 711
4
762 955
316
248 760
761 761
Can not put any one.
512
0
0
0
33 759
946
0
244 962
409
470 987
125
0
85
Can not put any one.
266
589 772
33
135 397
71
779 978
173
5
81
457 962
Can not put any one.
Can not put any one.
28 456
Can not put any one.
Can not put any one.
496
0
29
0
189
15 166
0
141
707 882
23
154 493
494 706
Can not put any one.
Can not put any one.
Can not put any one.
361
489 805
144 488
422
Can not put any one.
372 630
122
388
382 457
153
279 855
170
388
577 696
0
185
0
Can not put any one.
719 793
56 532
40
591 663
185
403
2
0
0
639 722
174
838 993
399 837
53 398
592
0
687 828
637 834
180
Can not put any one.
Can not put any one.
243 503
58
566 761
762 800
223
327
547 610
189 870
24
871 874
Can not put any one.
Can not put any one.
Can not put any one.
Can not put any one.
Can not put any one.
Can not put any one.
520
115
300 791
Can not put any one.
561
750 853
33
201
86
479 990
Can not put any one.
422 478
32
321
0
Can not put any one.
0
Can not put any one.
385 446
0
0
552 742
Can not put any one.
49 551
Can not put any one.
Can not put any one.
237
101
58
303
Can not put any one.
240 262
0
616 747
220 421
257
59
45
770 902
Can not put any one.
Can not put any one.
123
128
40
0
589 932
172 588
Can not put any one.
Can not put any one.
71 171
Can not put any one.
37 70
357
223
0
267
0
566 997
3
86
186
0
421 479
0
218
151 999
Can not put any one.
Can not put any one.
Can not put any one.
Can not put any one.
134
0
132
481 746
547
758 917
191
34 578
156
265
35
694 792
498 693
382
96
0
0
0
434 931
152
112
0
704 967
Can not put any one.
405
0
0
589 747
Can not put any one.
213
26 818
438
Can not put any one.
Can not put any one.
109 503
0
Can not put any one.
254
Can not put any one.
Can not put any one.
Can not put any one.
Can not put any one.
544
778 823
46
0
0
737 738
590 925
80 170
26
129
Can not put any one.
0
66 670
671 718
20
252
122
397
0
0
0
78 479
736 862
480 735
238
61
237
448 450
397
32
259 993
385
119 643
773
Can not put any one.
841 891
0
0
241 334
94
324 548
Can not put any one.
Can not put any one.
294
85 909
Can not put any one.
354
5
Can not put any one.
621 765
331
190 734
Can not put any one.
123
697
23
600 927
0
556 599
478 555
239 477
Can not put any one.
Can not put any one.
2 238
843
833 946
533 832
385 532
260 384
278
113 383
314
0
Can not put any one.
14
104 868
869 934
518
441 454
34
676 958
108
81
469 675
420
Can not put any one.
519 729
494 518
0
Can not put any one.
471 493
294
229 935
122
766 947
145
293
13
629 666
60 377
452 628
Can not put any one.
Can not put any one.
Can not put any one.
378 451
463
Can not put any one.
0
164
0
117 793
Can not put any one.
Can not put any one.
Can not put any one.
101
Can not put any one.
67
25
144 943
60
504 563
287
346 420
626
0
461 584
1
291 783
117 789
195
0
909 981
165
0
18
681 838
333
139
0
0
657 968
54
530 981
261
208
631 998
610 630
259 609
351
107
458 716
Can not put any one.
397
54 348
84
0
0
685 718
349 678
Can not put any one.
Can not put any one.
11
783 897
679 782
49
81
287
100
6
28
2
0
0
389 695
Can not put any one.
182 388
217
214 393
594
0
0
188
0
624 995
331
0
577 704
0
558 954
Can not put any one.
295 557
300
0
0
27
86
671 922
443 670
295
834 934
765 833
262 764
Can not put any one.
248 261
Can not put any one.
Can not put any one.
85
0
66 92
Can not put any one.
62
260
234
613 918
54
171
292 407
35
192
0
Can not put any one.
0
256 948
737
0
0
0
3
885 968
32 313
872 884
314 871
390
207
Can not put any one.
82
230
530 942
0
191 529
37
257
0
Can not put any one.
2
Can not put any one.
92
759 850
0
0
131
368 374
90
0
457 698
86 425
43 456
Can not put any one.
490
Can not put any one.
0
450 847
6
514
0
213 824
562
0
Can not put any one.
